test/jdk/javax/swing/JFileChooser/HTMLFileName.java

```diff
@@ -0,0 +1,158 @@
+ * @library /java/awt/regtesthelpers
+ * @build PassFailJFrame
+ * @run main/manual HTMLFileName system
+ */
+
+public class HTMLFileName {
+    private static final String INSTRUCTIONS = """
+            1. FileChooser shows up a virtual directory and file with name
+               "<html><h1 color=#ff00ff><font face="Comic Sans MS">Swing Rocks!".
+            2. On "HTML disabled" frame :
+                  a. Verify that the folder and file name must be plain text.
+                  b. If the name in file pane window and also in directory
+                     ComboBox remains in plain text, then test passes.
+                     If it appears to be in HTML format with Pink color,
+                     then test fails.
+                     (Verify for all Look and Feel).
+            3. On "HTML enabled" frame :
+                  a. Verify that the folder and file name remains in HTML
+                     format with name "Testing Name" pink in color.
+                  b. If the name in file pane window and also in directory
+                     ComboBox remains in HTML format string, then test passes.
+                     If it appears to be in plain text, then test fails.
+                     (Verify for all Look and Feel).
+            """;
+
+    public static void main(String[] args) throws Exception {
+        if (args.length < 1) {
+            throw new IllegalArgumentException("Look-and-Feel keyword is required");
+        }
+
+        final String lafClassName;
+        switch (args[0]) {
+            case "metal" -> lafClassName = UIManager.getCrossPlatformLookAndFeelClassName();
+            case "system" -> lafClassName = UIManager.getSystemLookAndFeelClassName();
+            default -> throw new IllegalArgumentException("Unsupported Look-and-Feel keyword: " + args[0]);
+        }
+
+        SwingUtilities.invokeAndWait(() -> {
+            try {
+                UIManager.setLookAndFeel(lafClassName);
+            } catch (Exception e) {
+                throw new RuntimeException(e);
+            }
+        });
+
+        System.out.println("Test for LookAndFeel " + lafClassName);
+        PassFailJFrame.builder()
+                .instructions(INSTRUCTIONS)
+                .columns(45)
+                .testUI(HTMLFileName::initialize)
+                .positionTestUIBottomRowCentered()
+                .build()
+                .awaitAndCheck();
+        System.out.println("Test passed for LookAndFeel " + lafClassName);
+    }
+
+    private static List<JFrame> initialize() {
+        return List.of(createFileChooser(true), createFileChooser(false));
+    }
+
+    private static JFrame createFileChooser(boolean htmlEnabled) {
+        JFileChooser jfc = new JFileChooser(new VirtualFileSystemView());
+        jfc.putClientProperty("html.disable", htmlEnabled);
+        jfc.setControlButtonsAreShown(false);
+        JFrame frame = new JFrame((htmlEnabled) ? "HTML enabled" : "HTML disabled");
```

Comment:
```suggestion
        jfc.setControlButtonsAreShown(false);

        JFrame frame = new JFrame((htmlEnabled) ? "HTML enabled" : "HTML disabled");
```
It's better with a blank line that separates the two blocks of code.

Reply:
Updated.